### PR TITLE
docs(router): NavLink wording improvements

### PR DESCRIPTION
**Pull request by Tobbe**
Just trying to improve the wording in our docs to make them easier to read and understand

### CODE REVIEW

File: docs/docs/router.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ import { NavLink, routes } from '@redwoodjs/router'
 const MainMenu = () =>
   <ul>
     <li>
-      <!-- When match "/" -->
+      <!-- Will match "/" -->
```
Comment (jtoar) [inline docs/docs/router.md:221]: The original wasn't clear, but I think the spirit of it is lost if we replace it with just "will". I think "when" was supposed to be short for something like `adds "activeLink" when the URL matches "/"`. Maybe we could do `will add "activeLink" when the URL matches "/"`?

File: docs/docs/router.md [hunk 2]
```diff
@@ -238,12 +238,12 @@ const MainMenu = () =>
   </ul>
 ```
 
-Alternatively, you can add the `activeMatchParams` prop to your `NavLink` to match the current URL **partially**
+The `activeMatchParams` prop can be used to control how query params are matched
```
Comment (jtoar) [inline docs/docs/router.md:241]:
```suggestion
The `activeMatchParams` prop can be used to control how query params are matched:
```

File: docs/docs/router.md [hunk 3]
```diff
@@ -260,7 +260,7 @@ const MainMenu = () => (
 
 > Note `activeMatchParams` is an array of `string` _(key only)_ or `Record<string, any>` _(key and value)_
 
-More granular match, `page` key only and `tab=tutorial`
+More granular match; needs to be on the tutorial tab (`tab=tutorial`) and have the `page` key specified
```
Comment (jtoar) [inline docs/docs/router.md:263]:
```suggestion
More granular match; needs to be on the tutorial tab (`tab=tutorial`) and have the `page` key specified:
```

**Review by jtoar — APPROVED** [review]
Nice! Reads like an improvement to me. Left one comment and two minor edits. All small things so feel free to address if you want but optional

File: docs/docs/router.md (hunk 1 above)
Comment (Tobbe) [reply docs/docs/router.md]: https://github.com/redwoodjs/redwood/pull/10401

**Pull request merged**